crates/agglayer-node/src/node.rs

```diff
@@ -138,7 +139,7 @@ impl Node {
             Epoch::BlockClock(cfg) => {
                 info!(
                     "Starting BlockClock with provider: {}",
-                    config.l1.ws_node_url
+                    sanitize_ws_url(config.l1.ws_node_url.as_str())
```

Comment:
Hmm maybe the config field could hold a custom type that wraps `Url` and defines the `Display` trait so it removes the sensitive parts. So we get this behaviour consistently when printing into a string.